DataConnectors/Gitlab/azuredeploy.json

```
@@ -0,0 +1,871 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "PlaybookName": {
+            "defaultValue": "Gitlab-Sentinel",
+            "type": "string"
+        },
+        "PersonalAccessToken": {
+            "type": "securestring",
+            "defaultValue": "<Personal Access Token>",
+            "metadata": {
+                "description": "The personal access token for gitlab instance with read access."
+            }
+        },
+        "ProjectIds": {
+            "type": "string",
+            "defaultValue": "<ProjectIDs>",
+            "metadata": {
+                "description": "The project id for gitlab projects to be audited."
+            }
+        },
+        "GroupIds": {
+            "type": "string",
+            "defaultValue": "<GroupIDs>",
+            "metadata": {
+                "description": "The group Id for gitlab groups to be audited."
+            }
+        },
+        "workspaceId": {
+            "type": "string"
+        },
+        "workSpaceKey": {
+            "type": "securestring"
+        }
+    },
+    "functions": [],
+    "variables": {
+        "AzureKeyVaultName": "[concat('kv-', parameters('PlaybookName'))]",
+        "AzureKeyVaultConnectionName": "[concat('kv-', parameters('PlaybookName'))]",
+        "azureloganalyticsdatacollector": "[concat('azureloganalyticsdatacollector-', parameters('PlaybookName'))]"
+    },
+    "resources": [
+        {
+            "name": "[variables('AzureKeyVaultName')]",
+            "type": "Microsoft.KeyVault/vaults",
+            "apiVersion": "2019-09-01",
+            "location": "[resourceGroup().location]",
+            "tags": {
+                "displayName": "Gitlab-Keyvault",
```

```suggestion
                "displayName": "GitLab-KeyVault",
```

Should this also indicate that it is for a sentinel data connector? I'm not sure if there is a convention for this.

```suggestion
                "displayName": "Sentinel-Data-Connector-GitLab-KeyVault",
```